kubernetes/linux/setup.sh

```diff
@@ -33,7 +33,7 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 
 #install oneagent - Official bits (3/14/2022)
 if [ "${ARCH}" != "arm64" ]; then
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/1.17.0/azure-mdsd_1.17.0-build.master.354_x86_64.deb" -O azure-mdsd.deb
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/testing%2Fmdsd%2F10062022/azure-mdsd_1.23.0-build.master.12_x86_64.deb" -O azure-mdsd.deb
```

Comment:
lets make sure to get the perf tests run by AMA team. once the perf tests are done and no perf regressions then we can proceed on taking the MDSD build and also we also need to scale test from our end to make sure no regressions.